ui/eslint.config.js

```diff
@@ -0,0 +1,41 @@
+import { createConfig, detectOpts } from '@esrf/eslint-config';
+import { globalIgnores } from 'eslint/config';
+
+const opts = detectOpts(import.meta.dirname);
+
+const config = [
+  globalIgnores([
+    'build/',
+    'release/',
```

Comment:
Wondering if this folder still needs to be excluded. Anyone knows how/when/if it's still generated?

Reply:
`release` yes, it was used back in the day but not any longer